hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMDatanodeProtocolServer.java

```diff
@@ -280,6 +280,22 @@ public static SCMRegisteredResponseProto getRegisteredResponse(
     return cmd.getProtoBufMessage();
   }
 
+  private String constructCommandAuditMap(List<SCMCommandProto> cmds) {
+    StringBuilder auditMap = new StringBuilder();
+    auditMap.append('[');
+    for (SCMCommandProto cmd : cmds) {
+      if (cmd.getCommandType().equals(deleteBlocksCommand)) {
+        auditMap.append("commandType:" + cmd.getCommandType());
+        auditMap.append(" No. of deleteTransactions:");
+        auditMap.append(cmd.getDeleteBlocksCommandProto().getDeletedBlocksTransactionsList().size());
+      } else {
```

Comment:
I guess some separator is also needed between commands, maybe between command attributes, too.  The goal is to most closely match previous output while ommitting unwanted details.